backend/app/models.py

```diff
@@ -112,3 +115,128 @@ class TokenPayload(SQLModel):
 class NewPassword(SQLModel):
     token: str
     new_password: str = Field(min_length=8, max_length=40)
+
+
+################################################################################
+##                                 Start Here!                                ##
+################################################################################
+
+# Shared properties
+class PatientBase(SQLModel):
+    full_name: str = Field(min_length=1, max_length=255)
+    description: str | None = Field(default=None, max_length=255)
+    email: EmailStr = Field(max_length=100)
+    phone_number: str = Field(max_length=20)
+    height: float = Field()
+    weight: float = Field()
+    gender: int = Field()
+    birth_date: Optional[datetime] = Field(sa_column=Column(
+        TIMESTAMP(timezone=True),
+        nullable=False, 
+    ))
+ 
+
+
+# Properties to receive on item creation
+class PatientCreate(PatientBase):
+    pass
+
+
+class PatientRegister(SQLModel):
+    email: EmailStr = Field(max_length=255)
+    password: str = Field(min_length=8, max_length=40)
+    full_name: str = Field(default=None, max_length=255)
+
+
+# Properties to receive on item update
+class PatientUpdate(PatientBase):
+    full_name: str | None = Field(default=None, min_length=1, max_length=255)  # type: ignore
+    description: str | None = Field(default=None, max_length=255)
+    email: EmailStr | None = Field()
+    phone_number: str | None  = Field(max_length=20)
+    height: float | None = Field()
+    weight: float | None = Field()
+    gender: int | None = Field()
+
+# Database model, database table inferred from class name
+class Patient(PatientBase, table=True):
```

Comment:
Consider which fields need to be unique, and enforce it in here.
I think eMail and phone number should be unique.


Look at userBase for reference.